offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -240,6 +242,16 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
 
     return generatedKeys[0]
   }
+  
+  private void removeCustomerAffiliations(Connection connection, int customerId, List<Integer> affiliationIds) {
+    String query = "DELETE FROM person_affiliation WHERE person_id = ? AND affiliation_id = ?"
+    
+    affiliationIds.each {affiliationId -> 
+      statement.setInt(1, customerId)
+      statement.setInt(2, affiliationId)
+      statement.execute()
+    }
+  }
```

Comment:
I think we should not remove the affiliation for a customer

Reply:
agreed, the code to remove it will be removed